config.w32

```diff
@@ -46,24 +43,13 @@ function mongodb_parse_libbson_version_file(inpath)
   return replacements;
 }
 
-function mongodb_parse_version_file(inpath, prefix)
+function mongodb_get_libmongocrypt_version()
```

Comment:
Changed this function to just return the string needed for `MONGOCRYPT_BUILD_VERSION`: https://github.com/mongodb/libmongocrypt/blob/40cc4eea42028e5cdf907b996d36aa2187c08441/src/mongocrypt-config.h.in#L22-L27